ansible/playbooks/roles/compatibility_layer/tasks/install_prefix.yml

```diff
@@ -135,6 +135,51 @@
   tags:
     - build_prefix
 
+- name: "Fix missing profile for arm64 - create directory"
+  ansible.builtin.file:
+    path: "{{ gentoo_prefix_path }}/var/db/repos/gentoo/profiles/prefix/linux/arm64"
+    mode: '0755'
+    state: directory
+  when: eessi_host_arch == "aarch64"
+  tags:
+    - build_prefix
+
+- name: "Fix missing profile for arm64 - eapi"
+  ansible.builtin.copy:
+    content: "5"
+    dest: "{{ gentoo_prefix_path }}/var/db/repos/gentoo/profiles/prefix/linux/arm64/eapi"
+    mode: 0644
+  when: eessi_host_arch == "aarch64"
+  tags:
+    - build_prefix
+
+- name: "Fix missing profile for arm64 - parent"
+  ansible.builtin.copy:
+    content: |
+      ../../../default/linux/arm64/17.0
+      ..
+    dest: "{{ gentoo_prefix_path }}/var/db/repos/gentoo/profiles/prefix/linux/arm64/parent"
+    mode: 0644
+  when: eessi_host_arch == "aarch64"
+  tags:
+    - build_prefix
+
+- name: "Fix missing profile for arm64 - make.defaults"
+  ansible.builtin.copy:
+    content: |
+      # Copyright 1999-2018 Gentoo Foundation
+      # Distributed under the terms of the GNU General Public License v2
+
+      ACCEPT_KEYWORDS="arm64 ~arm64"
+
+      # Not sure if this is required as well, seems to be the default:
+      # CHOST=aarch64-unknown-linux-gnu
+    dest: "{{ gentoo_prefix_path }}/var/db/repos/gentoo/profiles/prefix/linux/arm64/make.defaults"
+    mode: 0644
+  when: eessi_host_arch == "aarch64"
+  tags:
+    - build_prefix
+
```

Comment:
This has been merged upstream, so it's no longer necessary.

Reply:
True. Removed with https://github.com/EESSI/compatibility-layer/pull/177/commits/715823004458272f3d21ae16d72a6f0a7ade4b80